client/Brocfile.js

```diff
@@ -1,8 +1,11 @@
 /* global require, module */
 
 var EmberApp = require('ember-cli/lib/broccoli/ember-app');
+var mergeTrees = require('broccoli-merge-trees');
+var pickFiles = require('broccoli-static-compiler');
 
-var app = new EmberApp({storeConfigInMeta: false});
```

Comment:
I wanted to pull all configuration together in one place... i.e., using config/environment.js. Moved the storeConfigInMeta option there (https://github.com/Tahi-project/tahi/pull/816/files#diff-3cd1a229b31460d78e7e154670278c55R20)